### PR TITLE
COMMERCE-3912 As a logististics specialist, I want to create a shipment 1/2

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp [hunk 1]
```diff
@@ -93,8 +93,8 @@ CommerceAddress shippingAddress = commerceShipmentDisplayContext.getShippingAddr
 					'/commerce.commercecountry/get-shipping-commerce-countries',
 					{
 						companyId: <%= company.getCompanyId() %>,
-						end: -1,
-						start: -1,
+						shippingAllowed: true,
+						active: true,
```
Comment (dsitu) [inline modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp:97]: @riccardo-alberti I think we should sort these.

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp (hunk 1 above)
Comment (riccardo-alberti) [reply modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp:97]: hey @dsitu the service method is 
public List<CommerceCountry> getShippingCommerceCountries(
		long companyId, boolean shippingAllowed, boolean active) {

Comment (dsitu) [reply modules/apps/commerce/commerce-shipment-web/src/main/resources/META-INF/resources/shipment/address.jsp:97]: @riccardo-alberti ohhh ic. Yeah we should follow the original method order then. Thanks for clarifying!

**Review by dsitu — APPROVED** [review]

**Review by marco-leo — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**